src/main/java/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,62 @@
+package controller;
+
+import dto.cart.AddToCartRequestDto;
+import dto.cart.CartItemRequestUpdateDto;
+import dto.cart.ShoppingCartDto;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import model.User;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import service.ShoppingCartService;
+
+@RestController
+@RequestMapping("/api/cart")
```

Comment:
It's not necessary but you can add this property `server.servlet.context-path=/api` to application.properties and write all RequestMapping's without prefix `/api`